### ISSUE TITLE
Feature Request Set receiver Per Alert Condition/Callback

### ISSUE THREAD

**Issue by gruselglatz** [Graylog2/graylog2-server#3475]
Add the possibility to set the receivers per alert condition/callback.

We have to alert to different persons for different systems, and we dont want to create thousands of streams for each system.

## Expected Behavior
Set alert condition and callback and set receiver. always independent

## Current Behavior
All Conditions and Callbacks always get sent to all set receivers

**Comment by edmundoa** [comment]
Hi,

I'm not sure if understood the request correctly, but I think this is already fixed in the upcoming 2.2.0. Please take a look at the [RC release](https://www.graylog.org/blog/85-announcing-graylog-v2-2-0-rc-1) and reopen the issue if needed. The final version will be released shortly, in case you prefer to wait.

Thank you.

**Comment by gruselglatz** [comment]
@edmundoa maybe i have overseen it but i tried the new alert mechanism in 2.2.0 but i cant build what i wanted.

I want to send an alert when in the stream "test" more than x messages AND messages contain severity "C" are noticed to the person "D".

And when in the stream "test" more than x messages AND messages contain severity "A" are noticed to the person "B".

i haven't found the option to link conditions with each other.

**Comment by edmundoa** [comment]
No, that's currently not possible. Reopening.

**Comment by Kakuden** [comment]
How is the progress of this issues ?

**Comment by kanoblake** [comment]
I'd also like to see this issue addressed.  Here's another example use case:

I've got apache logs going into a stream that includes an httpd_time_to_serve_request field.  I would like:

1. A notification to Slack for the dev team when the average httpd_time_to_serve_request field goes over a specified value.
2. A notification to OpsGenie if the message count is < 1 in the past minute (ie. server is likely down)

If I have to create duplicate streams of the same apache data just to manage the possible alerts and conditions, they quickly get out of hand.

**Comment by kianris** [comment]
We have a use case that I think describes the same problem. Each user only needs to be notified on errors that occur in their features, so I add a feature with an error message "Error in feature ABCD" and I only want to get e-mails to my private e-mail for that condition.
Seems like this use case was missed in the new alerts system?

**Comment by BruiserT** [comment]
I was really hoping to see this added to the version 3.0.0 milestone list. This is a huge add for if your apart of an organization and don't want to create streams just for notifying different groups of people.

**Comment by rockrotem** [comment]
I totally agree with @BruiserT comment. You really need to change this behavior so the user can be notified based on condition and not based on stream.
I want to add multiple conditions on a specific stream but when notification triggered to get the only notification for the correct condition that triggers the alert. 

Do you plan to support it?

**Comment by Saif456** [comment]
Hi,

Is there any update on this issue I too want to add multiple conditions on a specific stream but when notification triggered to get the only notification for the correct condition that triggers the alert

Will it be supported?

**Comment by bernd** [comment]
@gruselglatz @Saif456 This is possible since Graylog 3.1.0 with the new alerting system. I am closing this issue now. Thank you!
